trade-finance-manager-api/src/v1/controllers/send-tfm-email.js

```diff
@@ -13,35 +13,37 @@ const sendTfmEmail = async (
     emailVariables,
   );
 
+  if (deal) {
   // update deal history
-  const dealId = deal._id; // eslint-disable-line no-underscore-dangle
-  const newHistoryObject = {
-    recipient: sendToEmailAddress,
-    templateId,
-    timestamp: now(),
-  };
-
-  let emailHistory = [];
-  if (deal.tfm?.history && deal.tfm.history.emails) {
-    emailHistory = deal.tfm.history.emails;
+    const dealId = deal._id; // eslint-disable-line no-underscore-dangle
```

Comment:
we don't need `// eslint-disable-line no-underscore-dangle` anymore

Reply:
done